### PR TITLE
Added `Back to Programs`  button

### LINKED ISSUE #1859: [Programs] Add "Back to Programs" button
**Current Behavior**
The individual open source internship pages (e.g.https://layer5.io/programs/gsoc/2019) do not have a Back to Programs button to help the user navigate back to the main programs page (https://layer5.io/programs).

#### Desired Behavior
A back button is present like that of the one on this page - https://layer5.io/community/members/jash-patel

Screen Shot 2021-05-15 at 11 13 03 AM

**Screenshots / Mockups**

Screen Shot 2021-05-15 at 11 09 32 AM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Fill-in the [Layer5 Community Member Form](https://layer5.io/newcomer) and join the [Layer5 Community Slack](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by amino19**
_Signed Off : Anshumaan_ anshumaankrprasad76@gmail.com

**Description**
Added ` <-- Back button` to Program page. 

This PR fixes #1859 

**Notes for Reviewers**
Not sure about work completed, pl give me more time to view on this issue.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/programs/Programs.style.js [hunk 1]
```diff
@@ -1,6 +1,29 @@
 import styled from "styled-components";
 
 export const ProgramsWrapper = styled.div`
+    margin: 4.5rem 0 3.125rem 0;
```
Comment (Tanuj22) [inline src/collections/programs/Programs.style.js:4]: This css should be in `ProgramsSingle.style.js` file

File: src/sections/Careers/Careers-Programs-single/index.js [hunk 2]
```diff
@@ -15,6 +15,14 @@ const ProgramsSingle = ({data}) => {
       />
       <div className="single-post-wrapper" id="programs">
         <Container>
+          <Row>
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/index.js:18]: To use `Row` and `Link` you need to import them first.

File: src/sections/Careers/Careers-Programs-single/index.js [hunk 3]
```diff
@@ -15,6 +15,14 @@ const ProgramsSingle = ({data}) => {
...
+              <div className="backBtn">
+                <Link to="careers/programs">
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/index.js:20]: the link should be `/careers/programs/`

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Thank you for this @amino19. Were you able to build to run the site locally? I ask because the build seems to fail (should be fixed after the changes are done)

File: src/sections/Careers/Careers-Programs-single/index.js [hunk 4]
```diff
@@ -1,5 +1,7 @@
 import React from "react";
+import {Link} from "gatsby";
 import { MDXRenderer } from "gatsby-plugin-mdx";
+import {Col, Container, Row} from "../../../reusecore/Layout";
 
 import { Container } from "../../../reusecore/Layout";
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/index.js:6]: Why are you importing this again? Remove this

File: src/collections/programs/ProgramsSingle.style.js [hunk 5]
```diff
@@ -1,6 +1,29 @@
 import styled from "styled-components";
```
Comment (Tanuj22) [inline src/collections/programs/ProgramsSingle.style.js:1]: Do not rename this file. No changes are needed in this file. The CSS needs to be added in this file https://github.com/layer5io/layer5/blob/master/src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js

**Review by Tanuj22 — CHANGES_REQUESTED** [review]

File: src/sections/Careers/Careers-Programs-single/index.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import React from "react";
+import {Link} from "gatsby";
 import { MDXRenderer } from "gatsby-plugin-mdx";
-
+import { Row } from "../../reusecore/Layout";
 import { Container } from "../../../reusecore/Layout";
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/index.js:5]: Add `Row` directly here

File: src/collections/programs/Programs.style.js (hunk 1 above)
Comment (Tanuj22) [inline src/collections/programs/Programs.style.js:4]: This css is no longer needed

File: src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js [hunk 7]
```diff
@@ -1,7 +1,30 @@
 import styled from "styled-components";
 
 const ProgramsPageWrapper = styled.div`
- margin-bottom: 30px;
+margin: 4.5rem 0 3.125rem 0;
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js:4]: The margin is probably going be messed up due to this. You need to also account for the previous `margin-bottom: 30px;`

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
@amino19 Try to run the site locally first ([instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)), and then make the changes. Then check out the changes locally in your web browser. Otherwise there may be issues each time

**Comment by l5io** [comment]
🚀 Preview for commit 2841b930b119ccedaba1a0dfba93ec0ba1a48b17 at: https://60a8bf508eed10c0e4d552b2--layer5ng.netlify.app

File: src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js (hunk 7 above)
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js:4]: Change to `margin-bottom : 3,125rem`

File: src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js [hunk 8]
```diff
@@ -1,7 +1,30 @@
...
+.backBtn {
+    font-weight: 600;
+    margin-bottom: 1rem;
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/ProgramsSingle.style.js:7]: Remove `margin-bottom: 1rem;` and add 
```
position: absolute;
top: 9.30rem;
z-index: 2
```

File: src/sections/Careers/Careers-Programs-single/index.js [hunk 9]
```diff
@@ -1,6 +1,8 @@
 import React from "react";
+import {Link} from "gatsby";
 import { MDXRenderer } from "gatsby-plugin-mdx";
-
+import { IoIosArrowDropleftCircle } from "react-icons/io";
+import { Row } from "../../../reusecore/Layout";
```
Comment (Tanuj22) [inline src/sections/Careers/Careers-Programs-single/index.js:5]: Just make this - 
```
import { Row, Container } from "../../../reusecore/Layout";
```

and then remove -
```
import { Container } from "../../../reusecore/Layout";
```

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Almost done

**Comment by l5io** [comment]
🚀 Preview for commit 41860935c947ca633a35b785d3becad653280cca at: https://60a8d4abf0e37cf37d10c8ce--layer5ng.netlify.app

**Review by Tanuj22 — APPROVED** [review]
Thank you @amino19

**Pull request merged**